go.work

```diff
@@ -3,8 +3,8 @@ go 1.22.5
 use (
 	.
 	./dev/tools/controllerbuilder
-	./experiments/kompanion
-	./experiments/kubectl-plan
+//	./experiments/kompanion
```

Comment:
Looks unrelated - split into another PR maybe?

Reply:
I think I was having a problem with the linter or make command.. Let me see what happens if I just drop this change.

Reply:
Fixed for now :D